Makefile

```diff
@@ -150,52 +149,9 @@ install: all
 			chmod 644 $$i; \
 		done; \
 	fi
-#       examples
-	for i in examples/syscalls \
-	         examples/serial/* examples/mpi/* examples/other/*; do \
-	    install -d $(LIBEXEC_INST)/$$i; \
-	    install -pm 644 -t $(LIBEXEC_INST)/$$i $$i/*; \
-	done
-	chmod 755 $(LIBEXEC_INST)/examples/serial/hello/hello.sh \
-	          $(LIBEXEC_INST)/examples/syscalls/pivot_root \
-	          $(LIBEXEC_INST)/examples/syscalls/userns \
-	          $(LIBEXEC_INST)/examples/*/*/*.sh
-	find $(LIBEXEC_INST)/examples -name Build -exec chmod 755 {} \;
-#       tests
-	install -d $(TEST) $(TEST)/run
-	install -pm 644 -t $(TEST) test/*.bats test/common.bash test/Makefile
-	install -pm 644 -t $(TEST)/run test/run/*.bats
-	install -pm 755 -t $(TEST) test/Build.*
-	install -pm 644 -t $(TEST) test/Dockerfile.* test/Docker_Pull.*
-	install -pm 644 -t $(TEST) test/*.patch
-	install -pm 755 -t $(TEST) test/make-auto test/make-perms-test
-	install -d $(TEST)/chtest
-	install -pm 644 -t $(TEST)/chtest test/chtest/*
-	chmod 755 $(TEST)/chtest/Build \
-	          $(TEST)/chtest/*.py \
-	          $(TEST)/chtest/printns
-	ln -sf ../../../bin $(TEST)/bin
-#       shared library tests
-	install -d $(TEST)/sotest $(TEST)/sotest/bin $(TEST)/sotest/lib
-	install -pm 755 -t $(TEST)/sotest test/sotest/libsotest.so.1.0 \
-	                                  test/sotest/sotest
-	install -pm 644 -t $(TEST)/sotest test/sotest/files_inferrable.txt \
-	                                  test/sotest/sotest.c
-	ln -sf ./libsotest.so.1.0 $(TEST)/sotest/libsotest.so
-	ln -sf ./libsotest.so.1.0 $(TEST)/sotest/libsotest.so.1
-	install -pm 755 -t $(TEST)/sotest/bin test/sotest/bin/sotest
-	install -pm 755 -t $(TEST)/sotest/lib test/sotest/lib/libsotest.so.1.0
-#       Bats (if embedded)
-	if [ -d test/bats/bin ]; then \
-	    install -d $(TEST)/bats && \
-	    install -pm 644 -t $(TEST)/bats test/bats/CONDUCT.md \
-	                                    test/bats/LICENSE \
-	                                    test/bats/README.md && \
-	    install -d $(TEST)/bats/libexec && \
-	    install -pm 755 -t $(TEST)/bats/libexec test/bats/libexec/* && \
-	    install -d $(TEST)/bats/bin && \
-	    ln -sf ../libexec/bats $(TEST)/bats/bin/bats && \
-	    ln -sf bats/bin/bats $(TEST)/bats; \
+#	install test suite and examples if the right python is found
+	if (command -v "$$(head -1 test/make-auto | sed -E 's/^.+ //')"); then \
```

Comment:
Don't we use this `command` business a second time somewhere? Should probably put it in a variable.